.github/CODEOWNERS

```diff
@@ -0,0 +1,231 @@
+/helmfile.d/values/networkpolicy/ @elastisys/goto-security
+/helmfile.d/values/opensearch/ @elastisys/goto-logging-stack
+/helmfile.d/values/podsecuritypolicies/ @elastisys/goto-security
+/helmfile.d/values/rclone/ @elastisys/goto-security
+/helmfile.d/values/thanos/ @elastisys/goto-monitoring-stack
+/helmfile.d/values/trivy/ @elastisys/goto-security
+/helmfile.d/values/userCRDs/ @elastisys/goto-security
+/helmfile.d/values/admin-namespaces-sc.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/admin-namespaces-wc.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/autoscaling-monitoring.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/calico-accountant.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/calico-default-deny.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/cert-manager.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/cluster-admin-rbac.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/dev-namespaces.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/dex.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/file-copier.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/ingress-nginx-probe-ingress.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/ingress-nginx.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/kubeapi-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-prometheus-stack-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-state-metrics-extra-resource-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kured.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/letsencrypt.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/metrics-server.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/minio.yaml.gotmpl @elastisys/goto-pipeline-qa-release
+/helmfile.d/values/node-local-dns.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/openstack-monitoring.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-alerts-sc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-blackbox-exporter-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-opensearch-exporter.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-user-alerts-wc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/s3-exporter.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/sc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/tekton.gotmpl @elastisys/goto-continuous-delivery
+/helmfile.d/values/user-alertmanager.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/user-rbac.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/velero-* @elastisys/goto-security
+/helmfile.d/values/wc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+
+#
+# /helmfile.d/
+#
+
+/helmfile.d/README.md @elastisys/goto-scripts
```

Comment:
```suggestion
/helmfile.d/README.md @elastisys/goto-public-docs 
```

Reply:
I do actually think this is a bit closer to scripts than public docs, so I will keep that.

Reply:
Fine by me!